wordle-web/pages/index.vue

```diff
@@ -75,5 +71,37 @@ function onKeyup(event: KeyboardEvent) {
   } else if (event.key.match(/[A-z]/) && event.key.length === 1) {
     game.value?.addLetter(event.key.toUpperCase());
   }
+
+}
+function calcAttempts(){
+  var attempts = 0;
+  if(game.value.gameState == GameState.Won){
+    attempts = game.value.guessIndex + 1;
+  }else{
+    attempts = game.value.guesses.length + 5;
+  }
+  return attempts;
 }
+function postScore(playerNameIn: string, attemptsIn: number, timeIn: number){
+  console.log("score data " + playerNameIn + " " + attemptsIn + " " + 0);
+  let postScoreUrl ="Score/UpdateScore";
+  Axios.post(postScoreUrl, {
+    playerName: playerNameIn,
+    attempts: attemptsIn,
+    time: timeIn
+  }).then((response) => {
+    console.log("response from api " + response.data + " " + response.status);
+  });
+}
+watch(() => game.value.gameState, (value) => {
+  if(value == GameState.Won || value == GameState.Lost){
+    if(userName === "guest"){
+      showUserNameDialog.value = true;
+      // it is not working I don't think it is updating the value on the app.vue page to show the dialog
+    }
+    
+    postScore(userName.value as string, calcAttempts(), 0);
```

Comment:
```suggestion
    postScore(userName.value as string, game.value.guessIndex + 1, 0);
```
If you remove calcAttempts() function